library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * Time-related-property <nowiki>[Estimates of how often a graphoelement is seen in the recording. (Source: Beniczky ea 2017, Table 6, Table 8.)]</nowiki>
+        ** Appearance-mode <nowiki> {suggestedTag=Property-not-possible-to-determine, suggestedTag=Random, suggestedTag=Repetitive, suggestedTag=Varying} [Describes how the non-ictal EEG pattern/graphoelement is distributed through the recording. Occurrence of the non-ictal EEG pattern / graphoelement can be Random, Repetitive or Varying. Random: occurring without any rhythmicity / periodicity, Repetitive: occurring at an approximately regular rate / interval (generally of 1 to several seconds). Variable: occurring sometimes rhythmic or periodic, other times random, throughout the recording. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
-        ** Discharge-pattern <nowiki>[Describes the organization of the EEG signal within the discharge (distinguish between single and repetitive discharges)]</nowiki>
-            *** Single-discharge-pattern <nowiki> {suggestedTag=Finding-incidence} [Applies to the intra-burst pattern: a graphoelement that is not repetitive; before and after the graphoelement one can distinguish the background activity.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Rhythmic-trains-or-bursts-discharge-pattern <nowiki> {suggestedTag=Finding-prevalence, suggestedTag=Finding-frequency} [Applies to the intra-burst pattern: a non-ictal graphoelement that repeats itself without returning to the background activity between them. The graphoelements within this repetition occur at approximately constant period.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Arrhythmic-trains-or-bursts-discharge-pattern <nowiki> {suggestedTag=Finding-prevalence} [Applies to the intra-burst pattern: a non-ictal graphoelement that repeats itself without returning to the background activity between them. The graphoelements within this repetition occur at inconstant period.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Fragmented-discharge-pattern
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Discharge-pattern <nowiki>[Describes the organization of the EEG signal within the discharge (distinguish between single and repetitive discharges). (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Single-discharge-pattern <nowiki> {suggestedTag=Finding-incidence} [Applies to the intra-burst pattern: a graphoelement that is not repetitive; before and after the graphoelement one can distinguish the background activity. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
```

Comment:
Can the tag names of the items under Discharge-pattern have the "-pattern" removed?  This seems unnecessary and redundant.